### PR TITLE
Show RML mod name in log messages

### LINKED ISSUE #6: Log messages from RML mods don't show the name of the mod
They should show the name of the mod.

### PR DESCRIPTION
**Pull request by Nytra**
Closes #6

### CODE REVIEW

File: MonkeyLoader.GamePacks.ResoniteModLoader/Util.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+namespace ResoniteModLoader;
+
+internal static class Util
+{
+    /// <summary>
+    /// Get the Logger for the mod from a stack trace.
+    /// </summary>
+    /// <param name="stackTrace">A stack trace captured by the callee</param>
+    /// <returns>The executing mod's Logger, or ModLoader.Logger if none found</returns>
+    internal static Logger GetLoggerFromStackTrace(StackTrace stackTrace)
```
Comment (Banane9) [inline MonkeyLoader.GamePacks.ResoniteModLoader/Util.cs]: you can just move this into the ResoniteMod or RmlMod class instead, no need for an extra util class. If you move it into the RmlMod class, the assembly map can be private too, as it's not accessed from anywhere else.

File: MonkeyLoader.GamePacks.ResoniteModLoader/ResoniteMod.cs [hunk 2]
```diff
@@ -32,13 +32,13 @@ protected ResoniteMod()
         /// Logs the given object as a line in the log if debug logging is enabled.
         /// </summary>
         /// <param name="message">The object to log.</param>
-        public static void Debug(object message) => ModLoader.Logger.Debug(() => message);
+        public static void Debug(object message) => Util.GetLoggerFromStackTrace(new(1)).Debug(() => message);
```
Comment (Banane9) [inline MonkeyLoader.GamePacks.ResoniteModLoader/ResoniteMod.cs]: You can wrap all these, so that => is on the next line and indented one block.

**Review by Banane9 — CHANGES_REQUESTED** [review]

**Comment by Banane9** [comment]
Everything else looks good 👍

**Comment by Nytra** [comment]
Changes applied

**Review by Banane9 — APPROVED** [review]

**Pull request merged**